tests/test_heart.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from opencmiss.zinc.context import Context
+from opencmiss.zinc.field import Field
+from opencmiss.zinc.result import RESULT_OK
+from scaffoldmaker.annotation.annotationgroup import AnnotationGroup
+from scaffoldmaker.meshtypes.meshtype_3d_heart1 import MeshType_3d_heart1
+from scaffoldmaker.utils.zinc_utils import evaluateFieldRange
+
+def assertAlmostEqualList(testcase, actualList, expectedList, delta):
+    assert len(actualList) == len(expectedList)
+    for actual, expected in zip(actualList, expectedList):
+        testcase.assertAlmostEqual(actual, expected, delta=delta)
+
+class HeartScaffoldTestCase(unittest.TestCase):
+
+    def test1_heart1(self):
```

Comment:
Test names should really start with `test_` this is what the test frameworks expect.

Reply:
Will fix.